build.cmd

```diff
@@ -1,42 +1 @@
-:: Optional batch file to quickly build with some defaults.
```

Comment:
I'm not only in favor of making this minimal, I'm actually conviced it's pointless. I will most likely totally remove `build.cmd` from the moq/moq repo. I think any .NET developer coming across a root `build.proj` will know what to do. Moreover, with corebuild.io, you even get dynamic help for its props/targets! See https://github.com/moq/moq/blob/master/build.proj

Reply:
Tadaa! It's gone completely now. 🍾

Reply:
It gets even better: It's perfectly possible to get rid of `build.proj` as well, meaning there will only be a `Moq.sln` left that MSBuild will picks up automatically (because it is the only project in that directory). So people can build Moq using a simple `msbuild`, `msbuild /t:Configure`, `msbuild /t:Test`, `msbuild /t:Pack`, whatever. It doesn't get any simpler.

Reply:
Not a fan of that, because it's not easy (even possible?) to add solution-level targets like `Configure` or `Test`. I'd keep `build.proj` since it's the thing that you can run to get reproducible builds that look and run (almost) exactly as what the CI builds, including all steps it runs, like testing and packaging.

People building from the cmdline aren't the same (I think) than those opening `Moq.sln` straight-away. So it's fine if they have different "paths". If the sln is inside a `src` dir, then both can find their sweet spot.

Reply:
> it's not easy (even possible?) to add solution-level targets like `Configure` or `Test`

It is. You can define them in `Before.Moq.sln.targets`.

You can also define dummy targets in `Directory.Build.props` that each solution project will "inherit" and be able to override as needed.

> I'd keep `build.proj` since it's the thing that you can run to get reproducible builds that look and run (almost) exactly as what the CI builds,